vetiver/handlers/base.py

```diff
@@ -79,3 +138,11 @@ def handler_predict(self, input_data, check_ptype):
             Prediction from model
         """
         ...
+
+
+@create_handler.register
+def _(model: base.VetiverHandler, ptype_data):
```

Comment:
Down the road I wonder if it would be helpful to standardize around separate language / argument names for...

* handlers
* VetiverModel objects
* fitted model objects (e.g. scikit learn model)

Reply:
wait I see what's happening here 😅

Reply:
(on this line of thought-- VetiverHandler is now called BaseHandler. It makes more sense with the file naming convention and is a bit more intuitive when subclassing :P hoping that slight change helps clarify things such as this!